python/lsst/ts/watcher/rules/mt_force_error.py

```diff
@@ -75,12 +86,33 @@ def get_schema(cls) -> dict[str, typing.Any]:
                     description: >-
                         Limit of the force error of axial actuator (N).
                     type: number
-                    default: 10.0
+                    default: 15.0
                 force_error_tangent:
                     description: >-
                         Limit of the force error of tangent link (N).
                     type: number
-                    default: 25.0
+                    default: 50.0
+                max_num_axial:
+                    description: >-
+                        Maximum number of the axial actuators that can have the
+                        excess force error.
+                    type: number
+                    default: 10
+                max_num_tangent:
+                    description: >-
+                        Maximum number of the tangent links that can have the
+                        excess force error.
+                    type: number
+                    default: 2
+                threshold_mtmount_acceleration:
+                    description: >-
+                        Threshold of the MTMount acceleration. The force error
+                        will not be checked if the acceleration is higher than
+                        this threshold. This is to avoid the wrong alarm from
+                        the overshoot of the control algorithm. The unit is
+                        deg/s^2.
+                    type: number
+                    default: 0.05
```

Comment:
You repeat all these default values in the configuration. Either remove them here or remove the default values from the ts_config_ocs config.

Reply:
Values are changed in the PR of ts_config_ocs. Thanks!